R/utils.R

```diff
@@ -92,3 +92,123 @@ browse_url <- function(path, chromote) {
     utils::browseURL(url)
   }
 }
+
+
+# =============================================================================
+# Random Ports
+# =============================================================================
+#
+# Borrowed from https://github.com/rstudio/httpuv/blob/main/R/random_port.R
+
+with_random_port <- function(
+  startup,
+  min = 1024L,
+  max = 49151L,
+  n = 5,
```

Comment:
I think the default should be at least 10.

Reply:
Done! 6a173fdadfb75c12d652dbeaec5aea93c0681b84